.ci/pitest-survival-check-xml.groovy

```diff
@@ -313,70 +311,90 @@ private static void printMutation(String flag, Mutation mutation) {
 private static Set<Mutation> setDifference(final Set<Mutation> setOne,
                                            final Set<Mutation> setTwo) {
     final Set<Mutation> result = new HashSet<>(setOne)
-    result.removeIf(mutation -> setTwo.contains(mutation))
+    result.removeIf { mutation -> setTwo.contains(mutation) }
     return result
 }
 
 /**
  * A class to represent the XML {@code mutation} node.
  */
 @EqualsAndHashCode(excludes = ["lineNumber", "unstable"])
-record Mutation(String sourceFile, String mutatedClass,
-                String mutatedMethod, String mutator,
-                String description, String lineContent, int lineNumber,
-                boolean unstable)
-        implements Comparable<Mutation> {
+final class Mutation implements Comparable<Mutation> {
```

Comment:
Can we use `@Immutable` to get a constructor as well, since this is just a simple data class?

Reply:
Done.